### PR TITLE
Bump to lts 20.19

### PR DESCRIPTION
**Pull request by agevelt**

### CODE REVIEW

File: src/Language/Elm/Pretty.hs [hunk 1]
```diff
@@ -36,7 +36,7 @@ import qualified Data.HashSet as HashSet
 import Data.List (sort, intersperse)
 import Data.Maybe (isNothing)
 import Data.String
-import Data.Text.Prettyprint.Doc
+import Prettyprinter
```
Comment (ocharles) [inline src/Language/Elm/Pretty.hs:39]: This needs the `prettyprinter` bound to be changed to `prettyprinter >= 1.6.0`

**Comment by ocharles** [comment]
I'll take a look when I'm home next week, I think I have merge rights

**Comment by ocharles** [comment]
I note that my previous review comment hasn't been addressed. What is it that people actually need? Are some of our bounds too restrictive? There's a whole bunch going on in this PR and it would be good to unpick

**Comment by ollef** [comment]
Folq folks (@agevelt @hanshenrik): do you want to keep these packages (elm-syntax, haskell-to-elm, servant-to-elm) under @folq? 

If not, I suggest we create a new 'haskell-to-elm' Github organization with some volunteers (preferably active users of the libraries) and move development there.

If you would like to keep them, can you please maintain them (keep track of and merge PRs etc) or ensure that someone who would like to maintain them has commit access on Github?

I am a maintainer for these packages on Hackage, and I can hand out maintainer bits to whoever ends up maintaining these libraries in the end (Folq or others). Notably haskell-to-elm is missing maintainers other than myself.

**Comment by agevelt** [comment]
This PR slipped my mind, sorry about that. 

Moving these to a different org is a good idea. Would you like to create the org and then I can transfer the repos @ollef?

Meanwhile I'll bump the `prettyprinter` bounds and merge this. I think I'll drop the version bump as well, and leave that to a separate PR.

**Comment by ollef** [comment]
Thanks @agevelt! Development has now moved to https://github.com/haskell-to-elm and I've invited past contributors there. There's still some setup to get sorted, but these libraries should hopefully soon be on track again.

**Comment by ollef** [comment]
That would be great @vladimirlogachev! Do you have the permissions to do it or is there anything I need to change?

**Pull request merged**